### PR TITLE
Store expenditure stake in the DB

### PR DESCRIPTION
**Pull request by jakubcolony**
https://github.com/JoinColony/colonyCDapp/pull/1958

### CODE REVIEW

File: src/handlers/expenditures/expenditureStakerPunished.ts [hunk 1]
```diff
@@ -1,11 +1,8 @@
-import { mutate } from '~amplifyClient';
-import {
-  UpdateExpenditureDocument,
-  UpdateExpenditureMutation,
-  UpdateExpenditureMutationVariables,
-} from '~graphql';
 import { ContractEvent } from '~types';
 import { getExpenditureDatabaseId, toNumber, verbose } from '~utils';
+import {} from '~utils/stakes';
```
Comment (bassgeta) [inline src/handlers/expenditures/expenditureStakerPunished.ts:3]: :eyes:

**Review by bassgeta — CHANGES_REQUESTED** [review]
looks good, works good, there's just an empty import there :D

**Review by bassgeta — APPROVED** [review]
all good now, nice one :ship:

**Review by iamsamgibbs — APPROVED** [review]
Reviewed with colonyCDapp#1958

**Pull request merged**